.stylelintrc.json

```
@@ -0,0 +1,3 @@
+{
+    "extends": "stylelint-config-standard-scss"
```

Nearly there! I believe we need both of them because the `scss` version only adds SCSS-specific rules, but doesn't contain the standard CSS rules.

```suggestion
    "extends": ["stylelint-config-standard", "stylelint-config-standard-scss"], 
```